ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/CurrentGroupParametersService.java

```diff
@@ -0,0 +1,17 @@
+package net.corda.v5.ledger.utxo;
+
+import net.corda.v5.membership.GroupParameters;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * {@link CurrentGroupParametersService} allows flows to access {@link GroupParameters} for further verification,
+ * for example, accessing to check if issued money is from authoritative party in flows.
+ */
+public interface CurrentGroupParametersService {
```

Comment:
I think this should be called `GroupParametersService`, at least based on what me and @relyafi decided a few weeks ago, let me know if that decision/idea has changed while i was away.

Reply:
Does it mean I should have created a new `GroupParametersService` rather than using the existing `CurrentGroupParametersService`?

Reply:
Can't you just rename this class to `GroupParametersService`?

Reply:
I can. What I'm not sure of is the service we are extending this interface in runtime repo is called CurrentGroupParametersService so my question is whether I have to rename it too or just this public API.

Reply:
wdyt @vlajos

Reply:
i think renaming the one in the runtime repo is fine.

Reply:
I agree with @lankydan that renaming both is the most consistent.